src/Components/PaymentField.res

```diff
@@ -105,7 +109,9 @@ let make = (
           name
           ?maxLength
           ?pattern
-          className={`Input ${inputClass} ${className} focus:outline-none transition-shadow ease-out duration-200`}
+          className={`${isSpacedInnerLayout
```

Comment:
Could you please move this condition outside

Reply:
Updated